pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2438,6 +2441,16 @@ public CompletableFuture<Void> checkClusterMigration() {
         }
     }
 
+    public boolean isReplicationBacklogExist() {
+        var replicators = getReplicators();
+        for (var val : replicators.values()) {
```

Comment:
what if it returns replicators null? can we do null check? and not sure if we want to use `var` here to avoid code inconsistency?

Reply:
also, can we use `replicator` instead of `val` to maintain naming consistency.

Reply:
Addressed